src/e2e/java/teammates/e2e/cases/e2e/AdminSearchPageE2ETest.java

```diff
@@ -177,17 +178,17 @@ private void verifyInstructorRowContent(InstructorAttributes instructor, Account
     }
 
     private String getExpectedInstructorHomePageLink(InstructorAttributes instructor) {
-        return instructor.isRegistered() ? createUrl(Const.WebPageURIs.INSTRUCTOR_HOME_PAGE)
-                .withUserId(instructor.googleId)
-                .toAbsoluteString()
-                : "";
+        String googleId = instructor.isRegistered() ? instructor.googleId : "undefined";
+        return createUrl(Const.WebPageURIs.INSTRUCTOR_HOME_PAGE)
+                .withUserId(googleId)
+                .toAbsoluteString();
     }
 
     private String getExpectedInstructorManageAccountLink(InstructorAttributes instructor) {
-        return instructor.isRegistered() ? createUrl(Const.WebPageURIs.ADMIN_ACCOUNTS_PAGE)
-                .withInstructorId(instructor.googleId)
-                .toAbsoluteString()
-                : "";
+        String googleId = instructor.isRegistered() ? instructor.googleId : "undefined";
```

Comment:
Same here

Reply:
Yep, I just didn't add a destructuring default value, so adding a default value prevents the stringified form of undefined from appearing.